.vscode/launch.json

```diff
@@ -136,7 +136,7 @@
             "type": "node",
             "request": "launch",
             "name": "Apollo Server (GraphQL ADR)",
-            "cwd": "${workspaceFolder}/doc/adr/0002/graphql-server-prototype",
+            "cwd": "${workspaceFolder}/doc/web-doc/adr/0002/graphql-server-prototype",
```

Comment:
I think this change is no longer applicable.

Reply:
Good catch! Fixed.